src/waveletTypes.jl

```diff
@@ -25,11 +25,11 @@ struct Morse <: ContinuousWavelets.ContWaveClass
     cf::Float64
 end
 
-N = 100; ga = 3; be = 1; cf = 1;
+ga = 3; be = 1; cf = 1;
 Morse() = Morse(ga,be,cf)
-class(::Morse) = "Morse"; name(::Morse) = "mors"; 
+class(::Morse) = "Morse"; name(::Morse) = "morse"; 
 vanishingmoments(::Morse) = 0; isAnalytic(::Morse) = true
-const mors = Morse()
+const morse = Morse()
```

Comment:
oh right, you should probably include `export morse` and `export Morse` somewhere (to deal with needing `ContinuousWavelets.morse` rather than just `morse`